portal/src/graphql/adminapi/UsersList.tsx

```diff
@@ -276,7 +287,9 @@ const UsersList: React.VFC<UsersListProps> = function UsersList(props) {
           const children =
             item.deleteAt != null ? (
               <FormattedMessage id="UsersList.cancel-removal" />
-            ) : item.isDisabled ? (
+            ) : item.anonymizeAt != null ? (
+              <FormattedMessage id="UsersList.cancel-anonymization" />
+            ) : item.isAnonymized ? null : item.isDisabled ? (
```

Comment:
When the user is anonymized by scheduled anonymization, the label is incorrect. In this case, user's `anonymizeAt` and `isAnonymized` are not empty.